### DISCUSSION TITLE
An Issue with FINN End to End examples

### DISCUSSION THREAD

**Discussion by MehdiRh17** [Xilinx/finn#485] [Q&A] [answered]
Hello! I am trying to run the notebook example (Xilinx FINN) . The problem is that when I run the 3rd part, I get this error:
AssertionError: vivado_hls not found in PATH

I already added my vivado path to the docker configuartion file. My vivado version is 2020.2 not 2020.1 ( or vitis hls 2020.2). Can the error be appeared becaused of this? I really appreciate it if you can help me with this. Thanks!

**Comment by sakshigoel155** [comment, marked as answer]
Hi 
I think this might be helpful, though it is about 2021.1 version but if 2020.2 also use vitis hls, it might be relevant.
https://github.com/Xilinx/finn/discussions/440

Regards,
Sakshi
